doc/development/deprecations.rst

```diff
@@ -9,6 +9,20 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
+* ``qml.from_qasm_file`` is an additional function to maintain that does not add much to the already existing
+  ``qml.from_qasm``. It is being deprecated, and users are directed to open the file themselves and load its
+  content using ``qml.from_qasm``.
+
+  - Deprecated in v0.36
+  - Will be removed in v0.37
+
+* The ``qml.load`` function is a general-purpose way to convert circuits into PennyLane from other
+  libraries. It is being deprecated in favour of the more specific functions ``from_qiskit``,
+  ``from_qasm``, etc.
+
+  - Deprecated in v0.36
+  - Will be removed in v0.37
+
```

Comment:
Wasn't this in a separate PR?

Reply:
This was, but that one hasn't been merged yet, awaiting one more approval. This PR branched off from the other one.

Reply:
Oops I didn't realize that when I do that, changes from both PRs are shown in this one. It'll clean up when this one is merged. https://github.com/PennyLaneAI/pennylane/pull/5312/